lib/game.ex

```diff
@@ -105,18 +105,29 @@ defmodule TicTacToe.Game do
 
   defp mark_board(current_board, {player_1, player_2}) do
     display_board(current_board)
-    if player_1.name =~ "Computer" do
-      Messager.turn_input_request(player_1.name, player_1.marker)
-      |> CliDisplay.write
-      computer_mark_cell(current_board, {player_1, player_2})
-    else
-      human_choose_cell(player_1)
-      |> mark_cell_if_available(current_board, {player_1, player_2})
-    end
+    cond do
+      player_1.name == "Computer" ->
```

Comment:
If we need to add more players in the future, we would have to open and modify this function which violates the open-closed principle. How can we mitigate this?